voltron.txt

```diff
@@ -36083,4 +36207,4 @@ dimwishlist:item=-1220931350#notes:A Puke Memorial 🤢🤢🤢 (Memorial is pre
 dimwishlist:item=-3325778512#notes:A Puke Memorial 🤢🤢🤢 (Memorial is pretty garbage even compared to Avalanche, both are outclassed severely by Delirium and Edgewise anyways)
  
 // rapid-fire machine gun
-dimwishlist:item=-3369545945#notes:Tempukal Clause 🤢🤢🤢 (Unless you *really* hate Hammerhead or Bane of Sorrow, this is redundant vs. better guns)
\ No newline at end of file
+dimwishlist:item=-3369545945#notes:Tempukal Clause 🤢🤢🤢 (Unless you *really* hate Hammerhead or Bane of Sorrow, this is redundant vs. better guns)
```

Comment:
The change here is I added a new line to the end of file. (which is pretty normal for most editors and tools.)

Any reason I shouldn't keep this as is? Or should I remove the new line?
![image](https://user-images.githubusercontent.com/6887340/72019639-162fa500-3238-11ea-8811-dd0d1f44b53e.png)